benches/benches/vm_set/mem.rs

```diff
@@ -164,13 +164,15 @@ pub fn run(c: &mut Criterion) {
     run_group_ref(
         &mut c.benchmark_group("pshh"),
         "pshh",
-        VmBench::new(op::pshh(full_mask)),
+        VmBench::new(op::pshh(full_mask))
+            .with_prepare_script(vec![op::pshh(full_mask); 10000]),
```

Comment:
Doesn't this affect the execution time?

Reply:
The same idea as with `aloc`. There is no difference in the current implementation, but maybe later, it can help us catch some mismatching in cost and results from sanity check.